apps/web/src/app/dashboard/[name]/settings/page.tsx

```diff
@@ -1,3 +1,13 @@
-export default function Settings() {
-  return <div className="flex flex-col gap-4">Settings</div>;
+"use client";
+
+import { usePathname, useRouter } from "next/navigation";
+import { useEffect } from "react";
+
+export default function SettingsView() {
+  const router = useRouter();
+  const pathname = usePathname();
+  useEffect(() => {
+    router.push(`${pathname}/general`);
+  });
+  return <></>;
```

Comment:
Another one for [this](https://github.com/users/hubcio2115/projects/3/views/1?query=is%3Aopen+sort%3Aupdated-desc&pane=issue&itemId=60319390)